cluster-autoscaler/apis/config/crd/autoscaling.x-k8s.io_provisioningrequests.yaml

```diff
@@ -227,6 +227,8 @@ spec:
     - jsonPath: .metadata.creationTimestamp
       name: Age
       type: date
+    deprecated: true
+    deprecationWarning: v1beta1 ProvisioningRequest is deprecated
```

Comment:
Let's make the warning `autoscaling.k8s.io/v1beta1 API is deprecated`

Ref:

- https://github.com/kubernetes/autoscaler/pull/5345/files#diff-b2539c9dfcc061e38b1961b5e52a8aed413750d74b71c018a3f6ca7dfc871efaR518